modin/config/__main__.py

```diff
@@ -42,18 +44,22 @@ def export_config_help(filename: str):
     filename : str
         Name of the file to export configs data.
     """
-    configs = pandas.DataFrame(
-        columns=[
-            "Config Name",
-            "Env. Variable Name",
-            "Default Value",
-            "Description",
-            "Options",
-        ]
-    )
+    configs_data = []
     for objname in sorted(globals()):
         obj = globals()[objname]
         if isinstance(obj, type) and issubclass(obj, Parameter) and not obj.is_abstract:
+            doc = obj.__doc__
+            description = []
+            indent = _get_indent(doc)
+            # remove indent
```

Comment:
Remove indent for better docs representation in the configs table.

Reply:
It looks like a different task.